### DISCUSSION TITLE
Error - AG 1.1.1 not working

### DISCUSSION THREAD

**Discussion by Umpalumpe** [apohl79/audiogridder#453] [Help] [answered]
I'm not getting AG 1.1.1 to work on my Mac OS Mojave. I'm trying to use it with Ableton 11 and use AG as a host plugin on the same mac to use all my CPU power on my computer, so I'm not using it with a slave or server.
So after scanning all of my plugins with the AudioGridderServer (app) and connect the plugin to my Server ID everything is fucked as you can see from the image attached.
So if anybody can help me or know a way to fix this - that would be wonderful?

![Skjermbilde 2021-03-29 kl  20 50 40](https://user-images.githubusercontent.com/81589852/112893804-6e6ff280-90db-11eb-8f97-3801db49e6a4.png)
![Skjermbilde 2021-03-29 kl  22 07 31](https://user-images.githubusercontent.com/81589852/112893852-7b8ce180-90db-11eb-8096-4830f101b515.png)

**Comment by apohl79** [comment, marked as answer]
You are using ver 1.1-beta10 on the plugin side with 1.1.1 server version. You have to upgrade the plugin to match the server version.

**Reply by Umpalumpe** [reply]
Thanks, what a rookie mistake. I just downloaded two of the server installer 1.1.1 and 1.1.0 and thought one of them were the plugins. It's time to wake up:)

**Reply by apohl79** [reply]
😉👍
